Makefile

```diff
@@ -32,9 +32,21 @@ upgrade: upgrade-piptools piptools ## upgrade requirement pins.
 	pip-compile requirements/test.in --rebuild --upgrade -o requirements/test.txt
 	pip-compile requirements/docs.in --rebuild --upgrade -o requirements/docs.txt
 	pip-compile requirements/dev.in --rebuild --upgrade -o requirements/dev.txt
-	# Delete django pin from test.txt so that tox can control Django version.
-	sed '/^[dD]jango==/d' requirements/test.txt > requirements/test.tmp
+	
+	# Delete django, drf pins from test.txt so that tox can control
+	# Django version.
+	sed -i.tmp '/^[dD]jango==/d' requirements/test.txt
+	sed -i.tmp '/^djangorestframework==/d' requirements/test.txt
+	rm requirements/test.txt.tmp
+	# Delete line "-e file:///local/path/to/edx-drf-extensions", which
```

Comment:
One-off comment: You can get rid of the `-e` removal stuff in the Makefile. That was there because of the `-e .` in `base.in`, but since you switched this repo over to use `load_requirements` instead, it's not necessary any more.